deck.gl/examples/pure-js/vue/carto2/package.json

```diff
@@ -8,11 +8,11 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@deck.gl/carto": "8.4.12",
+    "@deck.gl/carto": "8.5.8",
```

Comment:
^8.6.0

Reply:
Updated to 8.6.5. Sometimes there are breaking changes in minor versions, so I prefer to pin the example to a specific version.